### DISCUSSION TITLE
[Bug] .MPG files are stuttering with Audacious

### DISCUSSION THREAD

**Discussion by Levrifon** [audacious-media-player/audacious#85] [General]
Hello guys,

I recently discovered audacious for old soundfont from PSX, I now try also for an old game I played on PC but also available on CDI : Lost Eden, I downloaded the .MPG album and every tracks stutters, I tried to increase the buffer size but nothing change, is this a common bug ?

**Comment by Lascaille** [comment]
That sounds super annoying
